R/layers.R

```diff
@@ -60,7 +60,7 @@ layer_autoregressive <- function(object,
   args <- list(
     params = as.integer(params),
     event_shape = normalize_shape(event_shape),
-    hidden_units = as.integer(hidden_units),
+    hidden_units = as.list(hidden_units),
```

Comment:
Right! I think we might need to make sure this gets to be a list of integers?

Reply:
Yeah! Probably! Do we have a function for this?